packages/rocketchat-videobridge/client/views/videoFlexTab.js

```diff
@@ -0,0 +1,39 @@
+Template.videoFlexTab.helpers({
+
+});
+
+
+Template.videoFlexTab.onCreated(function() {
+	this.timeout = null;
+	this.autorun(() => {
+
+	});
+
+	// Opening a PR so we can do this via the https://meet.jit.si/external_api.js
+	$.getScript( "https://cdn.rawgit.com/geekgonecrazy/jitsi-meet/master/external_api.js" )
+		.done(function( script, textStatus ) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["textStatus" is defined but never used](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2700984385/issues/source?bid=3199634&fileBranchId=3342079#l14)